config/manager/manager.yaml

```diff
@@ -54,8 +54,8 @@ spec:
         # More info: https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted
         # Please uncomment the following code if your project does NOT have to work on old Kubernetes
         # versions < 1.19 or on vendors versions which do NOT support this field by default (i.e. Openshift < 4.11 ).
```

Comment:
I didn't notice this first time around: I guess this TODO comment can be removed now also?

Reply:
i thought about this, but still wanted to keep it just to be clear for anyone who saw it in the future.
but maybe it is not needed. i can do the clean up in another PR